src/config/config.ts

```diff
@@ -329,12 +329,8 @@ export class Config implements IConfig {
       if (b.pluginType === 'core' && a.pluginType !== 'core') {
         return 1
       }
-      // if a is a core plugin and b is not sort a first
-      if (a.pluginType === 'core' && b.pluginType !== 'core') {
-        return -1
-      }
-      // neither plugin is core, so do not change the order
-      return 0
```

Comment:
I was looking into why the unit tests are failing in CI and turns out we _do_ want this, even though the Array.prototype.sort() docs say this should be the same behavior.

Reply:
I went ahead and reverted it.

Reply:
Weird, but I will add it back in.